django_fsm_freeze/models.py

```diff
@@ -15,12 +15,13 @@ class FreezableFSMModelMixin(DirtyFieldsMixin, models.Model):
     class Meta:
         abstract = True
 
-    FROZEN_IN_STATES = ()
-    NON_FROZEN_FIELDS = ('state',)
+    FROZEN_IN_STATES: tuple = ()
+    FSM_STATE_FIELD_NAME: str = 'state'
```

Comment:
Is it possible to include a test to check if the configuration is correct? i.e. the value of `FSM_STATE_FIELD_NAME` is a field in the model and it's an `FSMField`?